TFT/src/User/API/flashStore.h

```diff
@@ -7,6 +7,7 @@ extern "C" {
 
 #include <stdbool.h>
 #include <stdint.h>
+#include "variants.h"
```

Comment:
I don't see any dependency on "variants.h" in flashstore.h. If it is needed in flashstore.c, please put it in the .c file